extensions/amp-timeago/0.2/use-intersect.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Let's move this file to `src/preact/use-intersect.js`, since it's very likely other components will use it.